setup.py

```diff
@@ -13,8 +13,10 @@
     long_description=open('README.rst').read(),
     url='https://github.com/mozilla/FoxPuppet',
     license='MPL2',
-    packages=['foxpuppet'],
+    packages=[
+        'foxpuppet',
+        'foxpuppet.browser'],
```

Comment:
Yeah, I looked into it a while back and think I determined it was due to https://bitbucket.org/pypa/wheel/issues/99/cannot-exclude-directory